modules/cluster-logging-collector-log-forwarding-supported-plugins-5-7.adoc

```diff
@@ -1,10 +1,15 @@
-// Text snippet included in the following assemblies:
+// Module included in the following assemblies:
 //
-//
-// Text snippet included in the following modules:
-//
-//
-:_content-type: SNIPPET
+// * logging/cluster-logging-external.adoc
+
+:_content-type: REFERENCE
+[id="cluster-logging-collector-log-forwarding-supported-plugins-5-7_{context}"]
+= Supported log data output types in OpenShift Logging 5.7
+
+Red Hat OpenShift Logging 5.7 provides the following output types and protocols for sending log data to target log collectors.
```

Comment:
- Not sure, but are there an attributes for the logging product name and/or version?
- Also, a nitpick that you don't have to fix in this PR, but `Red Hat` now needs a non-breaking space between the two words.

Reply:
There's a separate issue that I think Shreya is working on re the naming, we have other changes to do there to make it consistent in the doc set, so I'll probably leave this as is for now.